project/Dependencies.scala

```diff
@@ -29,4 +29,6 @@ object Dependencies {
   lazy val scalajHttp = "org.scalaj" %% "scalaj-http" % "2.3.0"
 
   lazy val cats = "org.typelevel" %% "cats-core" % "1.1.0"
+
+  lazy val jGit = "org.eclipse.jgit" % "org.eclipse.jgit" % "4.11.0.201803080745-r"
```

Comment:
This is a very old version. Check https://github.com/eclipse/jgit/releases, you might need to add a resolver for sonatype or something to get the new ones.